repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -126,37 +126,32 @@ class ReposeJettyServer(val nodeContext: AbstractApplicationContext,
 
         //Handle the Protocols and Ciphers
         //varargs are annoying, so lets deal with this using the scala methods
+        // From https://www.eclipse.org/jetty/documentation/9.4.x/configuring-ssl.html
+        // "When working with Includes / Excludes, it is important to know that Excludes will always win."
         import scala.collection.JavaConversions._
-        Option(ssl.getExcludedProtocols).foreach(xp => cf.addExcludeProtocols(xp.getProtocol.toList: _*))
-        Option(ssl.getIncludedProtocols).foreach { ip =>
-          if (ip.getProtocol.nonEmpty) {
-            cf.setIncludeProtocols(ip.getProtocol.toList: _*)
-          }
-
-        }
-        Option(ssl.getExcludedCiphers).foreach { xc =>
-          cf.addExcludeCipherSuites(xc.getCipher.toList: _*)
-        }
-
-        Option(ssl.getIncludedCiphers).foreach { ic =>
-          if (ic.getCipher.nonEmpty) {
-            cf.setIncludeCipherSuites(ic.getCipher.toList: _*)
-          }
-        }
+        // Clear Jetty defaults that were added in later v9.2.x releases.
+        cf.setIncludeProtocols()
+        cf.setExcludeProtocols()
+        cf.setIncludeCipherSuites()
+        cf.setExcludeCipherSuites()
+        Option(ssl.getIncludedProtocols) foreach { ip => cf.setIncludeProtocols(ip.getProtocol.toList: _*) }
+        Option(ssl.getExcludedProtocols) foreach { ep => cf.setExcludeProtocols(ep.getProtocol.toList: _*) }
+        Option(ssl.getIncludedCiphers) foreach { ic => cf.setIncludeCipherSuites(ic.getCipher.toList: _*) }
+        Option(ssl.getExcludedCiphers) foreach { ec => cf.setExcludeCipherSuites(ec.getCipher.toList: _*) }
```

Comment:
I wrote this comment at a bad time so it may have been hidden. I'll repeat it to make sure it is seen.

If it make it clearer, I believe you could replace these lines with lines like:
```
Option(ssl.getExcludedCiphers).map(_.getCipher.toList).foreach(cf.setExcludeCipherSuites)
```

Or to be even more explicit:
```
Option(ssl.getExcludedCiphers).map(_.getCipher).map(_.toList).foreach(cf.setExcludeCipherSuites)
```

Reply:
Updated to the more clear/less explicit version.

Reply:
Ok. That caused time mismatches. I reverted for now.

Reply:
Ah, I was afraid of that. Darn. I don't know if there's a good way to handle the typing weirdness for varargs. Oh well.